views/Products/Edit/index.tsx

```diff
@@ -57,10 +83,31 @@ const EditProductForm: FC<MutableProductFormProps> = ({ onSubmit, originalData }
         />
         {errors.description && <small className="text-red-500">{errors.description.message}</small>}
         <label className="text-size label">Añadir imagen del producto</label>
-        <label className="image font-visby">Añadir imagen</label>
-        {errors.imageUrl && <small className="text-red-500">{errors.imageUrl.message}</small>}
+        <div className={stylesInput['container-input']}>
+          <input
+            id="image-file"
+            type="file"
+            accept="image/*"
+            className={stylesInput['input-file']}
+            {...register('image')}
+            onChange={handleFile}
+          />
+          <label htmlFor="image-file" className={stylesInput.image}>
+            <figure>
+              <Image
+                width={imageUrl ? 300 : 40}
+                height={imageUrl ? 200 : 40}
+                objectFit="contain"
+                src={imageUrl || 'https://img.icons8.com/ios/100/image.png'}
+                alt="Imagen del evento"
```

Comment:
Change this alt, please, for example "Imagen del producto"

Reply:
done

Reply:
Thanks a lot! Good job